docker/Dockerfile

```diff
@@ -3,8 +3,9 @@ FROM ubuntu:20.04
 WORKDIR /sw
 
 COPY startup.sh startup.sh
-COPY edges.tsv.gz edges.tsv.gz
-COPY nodes.tsv.gz nodes.tsv.gz
+# TODO consider making the DKG code (i.e., "epi") and the date config variables
+RUN curl -o nodes.tsv https://askem-mira.s3.amazonaws.com/dkg/epi/build/2022-09-08/nodes.tsv
```

Comment:
alternative: check if there's a nodes.tsv file available (either in directory or via config) and only download if not.

Also: need to gzip these s3 files